business/definitionService.js

```diff
@@ -115,7 +116,10 @@ class DefinitionService {
     // If no tools participated in the creation of the definition then don't bother storing.
     // Note that curation is a tool so no tools really means there the definition is effectively empty.
     const tools = get(definition, 'described.tools')
-    if (!tools || tools.length === 0) return definition
+    if (!tools || tools.length === 0) {
+      await this.harvestService.harvest({ tool: 'package', coordinates })
```

Comment:
I don't think we need to `await` here, we're not going to use the results so queuing it up asynchronously would be fine.

Reply:
Just need to make sure any exceptions/rejected promises are caught and logged so they don't take the service down.